### PR TITLE
Update community documentation

### PR DESCRIPTION
**Pull request by adam-palazzo**
I added in a contributing and code of conduct. I also updated the README and package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +17,9 @@
   "bugs": {
     "url": "https://github.com/orbitdb/orbit-electron/issues"
   },
+  "homepage": {
+    "url": "https://github.com/orbitdb/orbit-electron/issues"
```
Comment (RichardLitt) [inline package.json]: I don't think this should point to /issues.

**Review by RichardLitt — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (adam-palazzo) [reply package.json]: It shouldn't and doesn't (now) :)

**Comment by adam-palazzo** [comment]
@RichardLitt Changes made

**Comment by RichardLitt** [comment]
Thanks.

**Pull request merged**